doc/classes/MultiplayerAPI.xml

```diff
@@ -118,7 +118,7 @@
 		<signal name="peer_connected">
 			<param index="0" name="id" type="int" />
 			<description>
-				Emitted when this MultiplayerAPI's [member multiplayer_peer] connects with a new peer. ID is the peer ID of the new peer. Clients get notified when other clients connect to the same server. Upon connecting to a server, a client also receives this signal for the server (with ID being 1).
+				Emitted when this MultiplayerAPI's [member multiplayer_peer] connects with a new peer. ID is the peer ID of the new peer. All peers get notified when a client connects to the same server. Upon connecting to a server, a client receives this signal multiple times, once for each other peer.
```

Comment:
The part about the server being sent as well should not be removed

Reply:
It is implied. Once for each other peer - all other clients and the server. Do you think it should be explicitly mentioned?

I don't see why, it's not like this signal needs special handling when emitted with the server's ID, in fact its the least valuable one since the server's ID is already known.

Reply:
I'd say it should, the fact that the server is a peer isn't clear or mentioned here, a user will probably not assume that

Reply:
Is this okay?